omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/RestHelperService.java

```diff
@@ -31,6 +32,9 @@
 	
 	List<Patient> getPatients(Collection<Integer> patientIds);
 	
+	@Authorized({ "View Patients" })
+	List<Patient> findPatients(String identifier, String searchType, boolean includeVoided);
```

Comment:
@rkorytkowski Done, but please take a look at the above test. Any idea why it stared to fail ?